pkgs/by-name/ni/nixfmt-rfc-style/generated-package.nix

```diff
@@ -22,7 +22,7 @@ mkDerivation {
   ];
   jailbreak = true;
   homepage = "https://github.com/NixOS/nixfmt";
-  description = "Opinionated formatter for Nix";
+  description = "An opinionated formatter for Nix";
```

Comment:
The reason this was removed was due to https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes:

> `meta.description` must ... [n]ot start with the definite or an indefinite article

Reply:
Ahh, that makes sense now, thanks! I updated it here and made a PR so it gets generated properly going forward: https://github.com/NixOS/nixfmt/pull/213